src/assets/languages/english.json

```diff
@@ -238,8 +240,32 @@
         "heading": "Prevention",
         "stopTheBreed": {
           "heading": "Eliminate all the mosquito breeding places by clicking over them ",
-          "alerts" : {
-            "title" : "Congratulations!! <br> You eliminated a "
+          "alerts": {
+            "title": "Congratulations!! <br> You eliminated a "
+          }
+        },
+        "stridesAgainst": {
+          "heading": "Click over the man to help moving it. You should reach the final point.",
+          "alerts": {
```

Comment:
When will this "Work in progress" alert appear?

Reply:
@Raxerz came here by mistake :sweat_smile: Removed it.